packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -190,11 +190,17 @@ class GoogleMapController {
   /// Also, refer [iOS](https://developers.google.com/maps/documentation/ios-sdk/style-reference)
   /// and [Android](https://developers.google.com/maps/documentation/android-sdk/style-reference)
   /// style reference for more information regarding the supported styles.
+  @Deprecated('Use GoogleMap.style instead.')
   Future<void> setMapStyle(String? mapStyle) {
     return GoogleMapsFlutterPlatform.instance
         .setMapStyle(mapStyle, mapId: mapId);
   }
 
+  /// Returns the last style error, if any.
```

Comment:
Should this say that it will get errors for the current mapid only?

Reply:
MapController is a per-map object; it's created with this ID.